src/ChurchCRM/Slim/Middleware/PublicCalendarAPIMiddleware.php

```
@@ -13,6 +19,50 @@ public function __invoke(Request $request, Response $response, callable $next)
         if (!SystemConfig::getBooleanValue("bEnableExternalCalendarAPI")) {
             return $response->withStatus(403)->withJson(["message" => gettext("External Calendar API is disabled")]);
         }
+        
+        $CAT = $request->getAttribute("route")->getArgument("CalendarAccessToken");
+        if (empty(trim($CAT))) {
+            throw new \Exception("Missing calendar access token");
```

please return a valid error message as we agreed to do a return $request->respose etc.. see

https://github.com/ChurchCRM/CRM/blob/98960fc94fcffa046d32a164d3fce5b6802d60c3/src/ChurchCRM/Slim/Middleware/AuthMiddleware.php#L30